helm/hpcc/templates/esp.yaml

```diff
@@ -75,6 +75,9 @@ data:
       dfuQueues:
 {{- include "hpcc.generateConfigDfuQueues" . | indent 6}}
 {{ include "hpcc.generateVaultConfig" . | indent 6 }}
+{{- if eq .me.application "eclwatch" }}
+ {{ include "hpcc.generateWarnings" (dict "root" .root) | indent 6 }}
```

Comment:
this might cause problems, it is indented by 1 space, and then by 6,.
I suggest:
```
{{- if eq .me.application "eclwatch" }}
 {{- include "hpcc.generateWarnings" (dict "root" .root) | nindent 6 }}
{{- end }}
```